libs/api/domains/law-and-order/src/lib/law-and-order.service.ts

```diff
@@ -0,0 +1,148 @@
+import { Injectable } from '@nestjs/common'
+import type { User } from '@island.is/auth-nest-tools'
+import { Locale } from '@island.is/shared/types'
+import {
+  CasesResponse,
+  Defender,
+  JudicialSystemSPClientService,
+  SubpoenaResponse,
+} from '@island.is/clients/judicial-system-sp'
+import { CourtCases } from '../models/courtCases.model'
+import { getSubpoena } from './helpers/mockData'
+import { CourtCase } from '../models/courtCase.model'
+import { Subpoena } from '../models/subpoena.model'
+import { Lawyers } from '../models/lawyers.model'
+import { PostDefenseChoiceInput } from '../dto/postDefenseChoiceInput.model'
+import { PostSubpoenaAcknowledgedInput } from '../dto/postSubpeonaAcknowledgedInput.model'
+import { SubpoenaAcknowledged } from '../models/subpoenaAcknowledged.model'
+import { mapDefenseChoice } from './helpers/mappers'
+
+@Injectable()
+export class LawAndOrderService {
+  constructor(private api: JudicialSystemSPClientService) {}
+
+  async getCourtCases(user: User, locale: Locale) {
+    const cases: Array<CasesResponse> | null = await this.api.getCases(
+      user,
+      locale,
+    )
+    if (cases === null) return null
+
+    const list: CourtCases = { items: [] }
+    const randomBoolean = Math.random() < 0.75
+
+    cases?.map((x: CasesResponse) => {
+      list.items?.push({
+        id: x.id,
+        acknowledged: randomBoolean,
+        caseNumber: x.caseNumber,
+        caseNumberTitle: x.caseNumber,
+        state: x.state,
+        type: x.type,
+      })
+    })
+
+    return list
+  }
```

Comment:
Ah this is just for developing, the service is not ready so I'm doing the magic 🪄  This will def be removed and replaced by actual data from the service

Reply:
And the return value, not quite sure which is considered best practice 🤔